python/setup.py

```diff
@@ -489,11 +499,12 @@ def has_ext_modules(self):
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
     ],
-    packages=setuptools.find_packages() if setup_spec.name == "ray" else [],
+    packages=setuptools.find_packages()
+    if setup_spec.type == SetupType.RAY else [],
```

Comment:
Could we make this a method fo SetupSpec? 

I.e.
```
class SetupSpec:
  def get_packages(self):
    if self.type == SetupType.RAY:
        return setuptools.find_packages()
    else:
        return []
```

Reply:
Good~